docs/app/pages/css/css-sections/icons/ux-icons/icon-preview/icon-preview.component.html

```diff
@@ -10,11 +10,18 @@
         [hideTriggers]="['click', 'clickoutside']">
 
     <ux-icon
+        *ngIf="iconset === 'ux-icons'"
         class="p-r-xs"
         [size]="size"
         [name]="name">
     </ux-icon>
 
+    <i
+        *ngIf="iconset === 'hpe-icons'"
+        class="hpe-icon p-r-xs"
```

Comment:
We have the following CSS rule currently:
````css
.btn .hpe-icon+* {
    margin-left: 4px;
}
````

Let's update this rather than use `p-r-xs`.

Reply:
updated style